fawkes/datastore/elasticsearch.py

```diff
@@ -149,3 +149,34 @@ def push_data_to_elasticsearch(fawkes_config_file = constants.FAWKES_CONFIG_FILE
                 print("[Error] push_data_to_elasticsearch :: Response is : ",
                       response.text)
             i += 1
+
+def query_from_elasticsearch(fawkes_config_file = constants.FAWKES_CONFIG_FILE, query_term="", format=constants.JSON):
+    fawkes_config = FawkesConfig(
+        utils.open_json(fawkes_config_file)
+    )
+    # For every app registered in app-config.json we
+    for app_config_file in fawkes_config.apps:
+        # Creating an AppConfig object
+        app_config = AppConfig(
+            utils.open_json(
+                app_config_file
+            )
+        )
+
+    if query_term == "":
+        endpoint = app_config.elastic_config.elastic_search_url + "_"+constants.SEARCH
```

Comment:
Did u run this through the linter ? I see there is no space between the operators and the operands here: `"_"+constants.SEARCH`